internal/worker/authenticationworker/utils.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package authenticationworker
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/juju/utils/v3/ssh"
+)
+
+const jujuEphemeralCommentPrefix = ssh.JujuCommentPrefix + "Ephemeral:"
+
+// ensureJujuEphemeralComment ensures that the key has a Juju ephemeral comment.
+// The comment is used to identify the ephemeral keys and flush them when the worker is restarted.
```

Comment:
"delete" reads better here since one might associate "flush" with flushing to disk.
```suggestion
// The comment is used to identify the ephemeral keys and delete them when the worker is restarted.
```